nuxeo-runtime/nuxeo-stream/src/test/java/org/nuxeo/lib/stream/tests/computation/TestStreamProcessor.java

```diff
@@ -598,6 +598,34 @@ public void testComputationPolicy() throws Exception {
 
     }
 
+
+    @Test
+    public void testComputationRecoveryPolicy() throws Exception {
```

Comment:
This test has since been removed on master by 35ef68a1c44e0b56aa55b6738ccf8f2b80595da7. What about it?

Reply:
Yes this test has been moved. I can follow 35ef68a to ease future backports.